src/components/ec-btn-dropdown/ec-btn-dropdown.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div
+    ref="popperReference"
+    class="ec-btn-dropdown"
+  >
```

Comment:
I guess is good to have this here as well
:data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-btn-dropdown` : 'ec-btn-dropdown'"